ratis-proto/src/main/proto/Raft.proto

```diff
@@ -175,6 +175,12 @@ message CommitInfoProto {
   uint64 commitIndex = 2;
 }
 
+message LogInfoProto {
+  TermIndexProto lastSnapshot = 1;
+  TermIndexProto applied = 2;
+  TermIndexProto committed = 3;
```

Comment:
Please add also `TermIndexProto lastEntry` (which is the last entry written to the log).

Reply:
yeah! i missed it, adding! Thanks Nicholas!

Reply:
added! Please take a final look, thanks!